concepts/05 Widgets/HtmlEditor/00 Overview.md

```diff
@@ -30,7 +30,7 @@ Follow the steps below to add the **HtmlEditor** to a page.
         <!-- ... -->
         <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_2/css/dx.common.css">
         <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_2/css/dx.light.css">
-        <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/20.2.2/js/dx-quill.min.js"></script>
+        <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/20.2.3/js/dx-quill.min.js"></script>
```

Comment:
If the Quill and DevExtreme version numbers are the same, you can use the `minor_20_2` placeholder instead of specifying the actual version.

Reply:
Did I get you right and I should make a link that looks as follows: 

`<script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_2/js/dx-quill.min.js"></script>`

If yes, won't users copy this link and get an error?

Reply:
No, the placeholder will be replaced with an actual link